src/css/relay.css

```diff
@@ -242,12 +242,15 @@ button.fx-relay-modal-close-button {
 
 .fx-relay-menu-remaining-aliases {
   font-size: 14px;
-  padding: 0 24px;
-  color: var(--relayInk70);
+  width: 260px;
+  padding: 12px 0;
+  background: var(--relayGrey20);
+  color: var(--relayGrey60);
   text-align: center;
-  margin-top: 8px;
-  margin-bottom: 24px;
-  font-weight: 600 !important;
+  font-weight: 600;
```

Comment:
This font-weight seems overly specific (_I'm expecting bold or not bold_). Do we have that specific font-weight available?

Reply:
Checking here: https://github.com/mozilla/fx-private-relay-add-on/blob/main/src/css/relay.css#L1-L26